packages/CodeGenLib/src/Config/config.ts

```diff
@@ -164,6 +164,10 @@ const sqlOutputConfigSchema = z.object({
    * ${flyway:defaultSchema}
    */
   convertCoreSchemaToFlywayMigrationFile: z.boolean().default(true),
+  /**
+   * If true, scripts that are being emitted via SQL logging that are marked by CodeGen as recurring will be SKIPPED. Defaults to false
+   */
+  omitRecurringScriptsFromLog: z.boolean().default(false)
```

Comment:
LGTM– this schema will parse the incoming value as a boolean, falling back to `false` if the parsed value is `undefined`. If the value is `null` or a number (e.g. `1` or `0`) or a string/object/etc it will throw a parsing exception.

Reply:
Should probably also add this to `mj.config.cjs` as well as its variants as desired (`docker.config.cjs` and `distribution.config.cjs`)